tests/phpunit/tests/block-bindings/postMetaSource.php

```diff
@@ -234,30 +234,4 @@ public function test_custom_field_not_exposed_in_rest_api_is_not_shown() {
 			'The post content should show the fallback value instead of the protected value.'
 		);
 	}
-
-	/**
-	 * Tests that meta key with unsafe HTML is sanitized.
-	 *
-	 * @ticket 60651
-	 */
-	public function test_custom_field_with_unsafe_html_is_sanitized() {
```

Comment:
I think it's fine to keep that test, too. It's better to stay on the safe side.

Reply:
I added it back in [this commit](https://github.com/WordPress/wordpress-develop/pull/6197/commits/193131fc0c3e6c69cfc4b3646f61b5e9878e51df).